libraries/AP_GPS/AP_GPS_SBF.cpp

```diff
@@ -121,6 +122,24 @@ AP_GPS_SBF::read(void)
                                     config_string = nullptr;
                                 }
                                 break;
+                            case Config_State::SBAS:
```

Comment:
Should canonicalise on either `ConfigStep::SBAS` or `config_state` - i.e. don't use step and state for the same thing

Pre-existing, of course.

Reply:
I see your point I think, but I'm going to defer that for scope creep reasons on this one.